src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -0,0 +1,415 @@
+      let department, jobTitle;
+
+      if (user.department) {
+        department = html`
+          <div class="department">${user.department}</div>
+        `;
+      }
+
+      if (user.jobTitle) {
+        jobTitle = html`
+          <div class="job-title">${user.jobTitle}</div>
+        `;
+      }
+      return html`
+        <div class="root">
+          <div class="default-view">
+            ${this.renderTemplate('default', { person: this.personDetails }) ||
+              html`
+                <mgt-person .personDetails=${this.personDetails} .personImage=${this.personImage}></mgt-person>
+                <div class="details">
+                  <div class="display-name">${user.displayName}</div>
+                  ${jobTitle} ${department}
+                  <div class="base-icons">
+                    ${this.renderIcons()}
+                  </div>
+                </div>
+              `}
+          </div>
+          <div class="additional-details-container" @click=${this._showAdditionalDetails}>
+            ${this.renderAdditionalDetails()}
+          </div>
+        </div>
+      `;
+    }
+  }
+
+  private renderIcons() {
+    const user = this.personDetails;
+    // tslint:disable-next-line: one-variable-per-declaration
+    let chat, email, phone;
+    if ((user as MicrosoftGraph.User).mailNickname) {
+      chat = html`
+        <svg xmlns="http://www.w3.org/2000/svg" width="21" height="22" viewBox="0 0 21 22" fill="none">
+          <path d="M15.173 6.20106H4.83114V4.69376H15.173V6.20106Z" fill="#3078CD" />
+          <path d="M4.83114 11.1766H11.3545V9.66933H4.83114V11.1766Z" fill="#3078CD" />
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M0.100098 2.39808C0.100098 1.18556 1.06777 0.199951 2.2857 0.199951H18.5145C19.7324 0.199951 20.7001 1.18556 20.7001 2.39808V20.2133C20.7001 20.9187 20.1975 21.4177 19.7305 21.6375C19.2562 21.8608 18.5192 21.9305 17.9793 21.3859L17.9397 21.346L15.1804 17.5802L15.162 17.5491C14.9874 17.2557 14.8758 17.2036 14.7939 17.176C14.6395 17.1241 14.4252 17.1152 13.9162 17.115C10.113 17.2743 5.24741 17.2819 2.42987 16.6132L2.36553 16.5979L2.30484 16.5717C1.77324 16.3418 1.21866 16.0979 0.815136 15.7451C0.347187 15.336 0.100098 14.8053 0.100098 14.1144V2.39808ZM2.2857 1.70725C1.91258 1.70725 1.6074 2.00562 1.6074 2.39808V14.1144C1.6074 14.3865 1.67852 14.4978 1.80722 14.6103C1.99344 14.7731 2.29809 14.9252 2.84155 15.1615C5.44205 15.761 10.0684 15.7681 13.8688 15.6084L13.8846 15.6077L13.968 15.6077C14.3859 15.6073 14.8562 15.6068 15.2743 15.7473C15.7779 15.9167 16.1388 16.2546 16.4378 16.746L19.0364 20.2925C19.0505 20.2891 19.0681 20.2833 19.0885 20.2738C19.1369 20.251 19.1718 20.2199 19.1892 20.1969L19.1928 20.192V2.39808C19.1928 2.00562 18.8876 1.70725 18.5145 1.70725H2.2857Z"
+            fill="#3078CD"
+          />
+        </svg>
+      `;
+    }
+    if ((user as MicrosoftGraph.User).mail) {
```

Comment:
You can use the `getEmailFromGraphEntity` function from the helpers that works with more than just user

Reply:
a20f0e1